.github/ISSUE_TEMPLATE/Patch-Version-Release-Checklist.md

```diff
@@ -0,0 +1,135 @@
+
+- Create a new tag
+<img width="502" alt="CleanShot 2023-09-05 at 22 06 24@2x" src="https://github.com/celery/celery/assets/4662342/69d7eb78-0dd1-4a37-9266-d718e2340fad">
+
+- Generate Markdown release notes
+<img width="389" alt="CleanShot 2023-09-05 at 22 13 39@2x" src="https://github.com/celery/celery/assets/4662342/83c6a4b6-1676-496e-8d75-5a6b10352b5b">
+
+- Copy the generated release notes.
+
+- Delete the draft release without publishing it.
+
+### 3.1 Changelog.rst
+Once you have the actual changes, you need to convert it to rst format and add it to the [Changelog.rst](https://github.com/celery/celery/blob/main/Changelog.rst) file. The new version block needs to follow the following format:
+```rst
+.. _version-x.y.z:
+
+x.y.z
+=====
+
+:release-date: YYYY-MM-DD HH:MM P.M/A.M TimeZone
+:release-by: Release Manager Name
+
+Changes list in RST format.
+```
+
+These changes will reflect in the [Change history](https://docs.celeryq.dev/en/stable/changelog.html) section of the documentation.
+
+### 3.2 Changelog PR
+The changes to the [Changelog.rst](https://github.com/celery/celery/blob/main/Changelog.rst) file should be submitted as a PR. This will PR should be the last merged PR before the release.
+
+## 4. Release
+### 4.1 Prepare releasing environment
+Before moving forward with the release, the release manager should ensure that bumpversion and twine are installed. These are required to publish the release.
+
+### 4.2 Bump version
+The release manager should bump the version using the following command:
+```bash
+bumpversion patch --new-version "x.y.z"
+```
+The changes should be pushed directly to main by the release manager.
+
+At this point, the git log should appear somewhat similar to this:
+```
+commit XXX (HEAD -> main, tag: vX.Y.Z, upstream/main, origin/main)
+Author: Release Manager
+Date:   YYY
+
+    Bump version: a.b.c → x.y.z
+
+commit XXX
+Author: Release Manager
+Date:   YYY
+
+    Added changelog for vX.Y.Z (#1234)
+```
+If everything looks good, the bump version commit can be directly pushed to `main`:
+```bash
+git push upstream main --tags
+```
+
+### 4.3 Publish release to PyPI
+The release manager should publish the release to PyPI using the following commands running under the root directory of the repository:
+```bash
+python setup.py clean build sdist bdist_wheel
+```
+If the build is successful, the release manager should publish the release to PyPI using the following command:
+```bash
+twine upload dist/celery-X.Y.Z*
+```
+
+> ⚠️ **Warning:** The release manager should double check that the release details are correct (project/version) before publishing the release to PyPI.
```

Comment:
Another thing we could consider adding here or any other docs that we should only yank recently released version

Reply:
Good point, I'll add it.

Reply:
Done